include/mata/nft/nft.hh

```diff
@@ -149,26 +149,11 @@ public:
     Nft& operator=(Nft&& other) noexcept;
 
     // Construct NFT from NFA
```

Comment:
I would update this comment too, maybe give a note to see the two functions in `builder`.